TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
Gossip might get relatively big...say 1M. Restarting from scratch means that a node will end up reading every newly connected peer's entire 1M map when connecting, despite likelihood of huge amounts of overlap. Not a high priority, but worth adding to the TODO list. It would be nice if a flapping node, for example, doesn't start killing bandwidth for its rack for absolutely no good reason.

Not sure what you mean by rescanning the whole thing? We can't scan anywhere to discover which replicas are the leaders of ranges. Are you referring to meta1 indexing records instead?